atk/Object.custom

```diff
@@ -61,16 +61,11 @@
 
 		static RefChildDelegate RefChildCallback;
 
-		[DllImport("libgobject-2.0-0.dll", CallingConvention=CallingConvention.Cdecl)]
-		static extern IntPtr g_object_ref (IntPtr handle);
-
 		static IntPtr RefChild_cb (IntPtr raw, int i)
 		{
 			try {
 				Atk.Object obj = GLib.Object.GetObject (raw, false) as Atk.Object;
 				Atk.Object child = obj.OnRefChild (i);
```

Comment:
I think we *should* be reffing unconditionally and it's the calling code's responsibility to ensure a non-null return value from `ref_child` is handled correctly. It's set as transfer ownership full so (ignoring the ref from the managed wrapper) we should be returning a reffed object which is then owned by native.

It is possible that we are adding too many refs, but I am not convinced (yet :p) that this patch is 100% correct.